packages/remix-dev/devServer_unstable/index.ts

```diff
@@ -196,6 +196,7 @@ export let serve = async (
         maxAge: "1y",
       })
     )
+    .use("*", express.static("public", { maxAge: "1h" }))
```

Comment:
I believe the public directory is configurable right? Maybe this should be dynamic based on the config?

Reply:
the Remix assets build directory (e.g. `public/build/`) is configurable, but I don't think we have config for `public/` itself.

Reply:
Ok we can make this configurable for dev. Shouldn't need to be set in prod since you already have control of your app server, but in dev we need to make sure we play nicely with any setup.

Reply:
`--public-directory` / `unstable_dev.publicDirectory` option added!